frontend/src/stores/search.ts

```diff
@@ -149,6 +149,11 @@ export const useSearchStore = defineStore("search", {
     state.recentSearches = useStorage<string[]>("recent-searches", [])
   },
   getters: {
+    searchStarted(state) {
+      return state.strategy === "default"
+        ? state.searchTerm.length > 0
+        : state.collectionParams !== null
+    },
```

Comment:
Nit: I feel like a comment could be useful to explain what this behavior is for. So basically `searchStarted` is a boolean used in calculating `canLoadMore`? and `searchStarted` represents that on standard views, there is a search term, and on collection pages, that the required params are present? Just making sure I follow :smile:

Reply:
I removed this in favor of `fetchState.hasStarted` in the media store, as the purpose of that property is to see if the fetching was started.